roles/edpm_frr/tasks/install.yml

```diff
@@ -0,0 +1,51 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Create persistent directories
+  ansible.builtin.file:
+    path: "{{ item.path }}"
+    state: directory
+    setype: "{{ item.setype }}"
+    mode: "{{ item.mode }}"
+  with_items:
+    - { 'path': /var/log/containers/frr, 'setype': container_file_t, 'mode': '0750' }
+    - { 'path': {{ edpm_frr_config_basedir }}, 'setype': container_file_t, 'mode': '0750' }
```

Comment:
you need to escape var with double quote. [Here](https://logserver.rdoproject.org/97/97/565c49537cda25203efd43333d3fecea47d20b0c/github-check/edpm-ansible-github-rdo-integration-centos-8-crc-singlenode-centos-9-external-compute/a7e2363/controller/controller/pod/deploy-external-dataplane-compute-6spks-logs.txt)'s the log.